client/components/payment-activity/style.scss

```diff
@@ -142,10 +140,48 @@
 			@include amount-styles;
 		}
 
+		&__tooltip {
+			.wcpay-inline-notice.components-notice {
+				margin-top: 8px;
+				margin-right: 0;
+				padding: 8px;
+				color: #044b7a;
+
+				.components-notice__content {
+					font-size: 11px;
+					line-height: 16px;
+					white-space: pre-wrap;
+					margin: 0;
+				}
+			}
+		}
+	}
+
+	#wcpay-payment-activity-data {
```

Comment:
It is a good practice to use class selector instead of id, to maintain consistency.

Reply:
Maybe increasing the hierarchy can help here? That's the usual practice I've seen.

Reply:
Thanks for the great hints, @naman03malhotra  🙌 

I have used a class to do what this snippet does : 27161d2 . Hope this is fine.